backend/app/services/mcp_providers/core/config.py

```
@@ -2,151 +2,23 @@
-# All built-in providers
-# Note: Providers are now auto-discovered from providers/ directory
+# Kept for callers that still import the legacy built-in provider list.
+# Providers are registered through auto-discovery.
 BUILTIN_PROVIDERS = []
+
+__all__ = [
+    "BAILIAN_CONFIG",
+    "MODELSCOPE_CONFIG",
+    "MCPROUTER_CONFIG",
+    "BUILTIN_PROVIDERS",
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)